src/pankajBhatt/Assignment14_5_Sep2022.java

```diff
@@ -0,0 +1,42 @@
+/*Assignment - 14 : 5th Sep'2022
+
+1. We need to do sum of first 5 prime number, using return keyword
+output: (2 + 3 + 5 + 7 + 11) = 28*/
+
+package pankajBhatt;
+
+public class Assignment14_5_Sep2022 {
+	int count = 0;
+	static int sum = 0;
+
+	boolean isPrime(int num) {
+		int index = 2;
+		boolean flag = true;
+
+		while (index <= num / 2) {
+			if (num % index == 0) {
+				flag = false;
+				break;
+			}
+			index++;
+		}
+		if (flag) {
+			count++;
+			sum = sum + num;
+		}
+		return true;
+	}
+
+	void firstFivePrimeNumbers(int number) {
+		int num = 2;
+		while (count < number) {
+			isPrime(num);
```

Comment:
line 23 to 26 code should be here, based on flag you should decide to do sum or not

if (flag) {
			count++;
			sum = sum + num;
		}